setup.py

```diff
@@ -34,9 +34,11 @@
     platforms=['linux'],
     packages=[
         'piker',
+        'piker.cli',
```

Comment:
Yeah, more I think about this it shouldn't be the cause of this import error since internally the packages use relative import syntax.

Not sure what's up..

Reply:
Ahh, I think it's that you're pip installing the master branch which is broken... If you were to make a `virtualenv` and install from this branch it would likely work.

Reply:
Also we should probably just use [`find_packages()`](https://setuptools.readthedocs.io/en/latest/setuptools.html#id8) for now.